### DISCUSSION TITLE
É possível de fato gerar notas fiscais eletronicas com essa aplicação?

### DISCUSSION THREAD

**Discussion by Joao-SDAVieira** [TadaSoftware/PyNFe#364] [Q&A]
Sou novo em programação e queria saber se de fato com esse pacote é possível gerar notas fiscais, estou querendo fazer uma automação de geração de notas fiscais, não entendi muito bem o pacote pela documentação pois não fala muito sobre como funciona e tudo mais...

**Comment by leogregianin** [comment]
@Joao-SDAVieira dá uma olhada no wiki: https://github.com/TadaSoftware/PyNFe/wiki tem vários exemplos de todas as operações de iteração dessa lib com os webservices.
